packages/deployed-backend-client/src/backend_output_client.ts

```diff
@@ -22,6 +23,50 @@ export class DefaultBackendOutputClient implements BackendOutputClient {
       this.amplifyClient
     ).getStrategy(backendIdentifier);
     const output = await outputFetcher.fetchBackendOutput();
+
+    if (
+      output['AWS::Amplify::Storage'] &&
+      Object.keys(output['AWS::Amplify::Storage'].payload).length >= 4
```

Comment:
- [ ] need to double check if `Object.keys(output['AWS::Amplify::Storage'].payload).length >= 4` or `>=2`